### DISCUSSION TITLE
是否有命令可以查询仓库里有哪些版本的包

### DISCUSSION THREAD

**Discussion by MichaelWU0726** [xmake-io/xmake#2076] [Q&A]
我是用`xmake require --search`命令返回的只有最新版本的包，而不是返回多个版本，希望可以提供像pypi这样的网页可以查看包的所有版本。

另外如何查询本地已下载/安装的包？我使用`xmake require --list`只是返回项目需要的包，并不是返回本地缓存安装的包列表。

官网目录是否可以设计成带有展开/折叠按钮的目录样式？目前官网目录带的短横线是个静态符号，无法手动折叠或者展开，很不方便。

**Comment by waruqi** [comment]
> 希望可以提供像pypi这样的网页可以查看包的所有版本。

https://xrepo.xmake.io/#/packages/linux?id=cjson-linux

> 另外如何查询本地已下载/安装的包？我使用xmake require --list只是返回项目需要的包，并不是返回本地缓存安装的包列表。

xrepo scan

> 官网目录是否可以设计成带有展开/折叠按钮的目录样式？目前官网目录带的短横线是个静态符号，无法手动折叠或者展开，很不方便。

不支持，用的 docsify 文档框架，你应该到 他们那边 提 issues
